stable/artifactory/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [9.5.4] - Jun 25, 2020
+* Notex.txt update - add --namespace parameter
```

Comment:
typo - Please change to `Notes` instead of `Notex`

Reply:
good catch, thx